source/video_core/src/video_core/vulkan/renderer.hpp

```diff
@@ -5,6 +5,7 @@
 #include "awaitable.hpp"
 
 #include <vulkan_utils/memory_types.hpp>
+#include <optional>
```

Comment:
Was this needed because the current build of the main branch is already failing for you or was it needed for another reason?

Reply:
Indeed the current build of the main branch is failing. There was other stuff I had to add, but `<optional>` was needed on two of my machines (both using PopOS).

Reply:
Yeah sounds good then. Feel free to submit any other fixes you might have in separate PRs even if they end up being tiny changes.